### PR TITLE
feat: [CDS-44813]: Cypress Test for Freeze Studio

### PR DESCRIPTION
**Pull request by ishantmehtaharness**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: cypress/integration/72-freze-windows/ProjectLevelFreeze.spec.ts [hunk 1]
```diff
@@ -61,6 +60,100 @@ describe('Project Level Freeze Creation', () => {
...
+    cy.get('li.Select--menuItem').should('have.length', 3).as('envTypeMenuItem')
+    cy.get('@envTypeMenuItem').eq(0).should('contain.text', 'All Environments').as('typeAllEnv')
+    cy.get('@envTypeMenuItem').eq(1).should('contain.text', 'Production').as('typeProd')
+    cy.get('@envTypeMenuItem').eq(2).should('contain.text', 'Pre-Production').as('typePreProd')
+
+    cy.get('@typeProd').click({ force: true })
+
+    // Select Service
+    cy.get('input[name="entity[0].Service"]').should('be.visible').click()
+    cy.get('li.MultiSelect--menuItem').should('have.length', 2).as('serviceTypeMenuItem')
```
Comment (vikrant-harness) [inline cypress/integration/72-freze-windows/ProjectLevelFreeze.spec.ts:115]: Try not to use classes to grab elements if possible

File: cypress/integration/72-freze-windows/ProjectLevelFreeze.spec.ts [hunk 2]
```diff
@@ -61,6 +60,100 @@ describe('Project Level Freeze Creation', () => {
...
+    cy.get('button span.bp3-icon-tick').click()
+    cy.wait(500)
+    cy.get('[data-testid="config-edit-mode_1"]').should('have.length', 0)
+    cy.get('[data-testid="config-view-mode_0"]').should('have.length', 1)
+    cy.get('[data-testid="config-view-mode_1"]').should('have.length', 1)
+
+    // Go To YAML - validate YAML
+    cy.contains('div[data-name="toggle-option-two"]', 'YAML').should('be.visible').click()
+    cy.wait(500)
+    cy.get('.view-lines div').should('have.length', 22)
```
Comment (vikrant-harness) [inline cypress/integration/72-freze-windows/ProjectLevelFreeze.spec.ts:132]: Is checking number of lines necessary here? if we are checking all the required fields?

**Review by vikrant-harness — APPROVED** [review]
Some minor comments

File: cypress/integration/72-freze-windows/ProjectLevelFreeze.spec.ts (hunk 2 above)
Comment (ishantmehtaharness) [reply cypress/integration/72-freze-windows/ProjectLevelFreeze.spec.ts:132]: Yes, I want to make sure, there is nothing else rendered on YAML

File: cypress/integration/72-freze-windows/ProjectLevelFreeze.spec.ts (hunk 1 above)
Comment (ishantmehtaharness) [reply cypress/integration/72-freze-windows/ProjectLevelFreeze.spec.ts:115]: I see, this is used at 10 other areas in the cypress folder
`Select--menuItem`
Please share if you have any other good alternate way to test this. Will use that in a follow up PR today.

**Review by kapilkumawat86 — APPROVED** [review]

**Pull request merged**